### PR TITLE
Explicit context in example

### LINKED ISSUE #901: ART-ART: Explicit context in the Send in example in 3.1.1
Consider showing an explicit context in the Send in the example in 3.1.1 - it
would make the reader guess less to wait to introduce implicit contexts later.

### PR DESCRIPTION
**Pull request by mwelzl**
Resolves #901.

Is this really what was being asked for here?  Just to create a MessageContext object? Or am I missing something?

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -440,6 +443,7 @@ C.Send(messageDataRequest)
 // Only receive complete messages
 C.Receive()
 //---- Ready event handler for any Connection C end ----
+messageContext := NewMessageContext()
```
Comment (abrunstrom) [inline draft-ietf-taps-interface.md:446]: This part did not change so why is this now needed to create the messageContext?

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -389,10 +389,13 @@ Listener -> ConnectionReceived<Connection>
 // Only receive complete messages in a Conn.Received handler
 Connection.Receive()
 
+messageContext := NewMessageContext()
```
Comment (abrunstrom) [inline draft-ietf-taps-interface.md:392]: Is this in the right place? Should it not be in the event handler where you send the message?

File: draft-ietf-taps-interface.md [hunk 3]
```diff
@@ -389,10 +389,13 @@ Listener -> ConnectionReceived<Connection>
...
 Connection -> Received<messageDataRequest, messageContext>
 
 //---- Receive event handler begin ----
-Connection.Send(messageDataResponse)
+messageContext.add(msgPrio, 200)
+messageContext.add(final, true)
+Connection.Send(messageDataResponse, messageContext)
```
Comment (abrunstrom) [inline draft-ietf-taps-interface.md:398]: Where does the messageDataResponse come from. I guess the example does not show everything? But it gets a bit confusing when the messageContext is filled explicitly.

File: draft-ietf-taps-interface.md [hunk 4]
```diff
@@ -389,10 +389,13 @@ Listener -> ConnectionReceived<Connection>
 // Only receive complete messages in a Conn.Received handler
 Connection.Receive()
 
+messageContext := NewMessageContext()
 Connection -> Received<messageDataRequest, messageContext>
```
Comment (tfpauly) [inline draft-ietf-taps-interface.md:393]:
```suggestion
Connection -> Received<messageDataRequest, receiveMessageContext>
```

File: draft-ietf-taps-interface.md (hunk 3 above)
Comment (tfpauly) [reply draft-ietf-taps-interface.md:398]: sendMessageContext := NewMessageContext()
sendMessageContext.add(msgPrio, 200)
sendMessageContext.add(final, true)
Connection.Send("response", sendMessageContext)

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-taps-interface.md:446]:
```suggestion
```

**Pull request closed without merging**